useful_resources/sample_code/README.md

```diff
@@ -0,0 +1,20 @@
+# Sample Scripts
+
+The scripts in this folder perform data operations, typically moving data between one connector and another. To perform the same operation, all you should need to do is change the configuration variables in the `Configuration Variables` section of each script. You can also use these scripts as a jumping off point to create new scripts doing similar opeations.
+
+If you can't find the script you want here, you can create an issue in the tracker with the label [sample script request](). Please put in as much detail as possible what you would like the script to do.
+
+If you have a script you'd like to add, you have two options. You can create an issue in the tracker with the contents of your script, and add the label [sample script to add](). Or you can add the script yourself.
+
+If you wish to add the script yourself, please use the `template_script.py` file so that your contribution's structure is consistent with all the other scripts.  Please keep most comments as these are directed at the eventual user of your script, but delete any comments labeled "//To Script Writer//" once you understand their advice.
+
+Please also add your new script to the table below.
+
+# Existing Scripts
+
+| File Name   | Brief Description |  Connectors Used   |
+| ----------- | ----------- |  ----------- |
+| apply_activist_code.py  | Gets activist codes stored in Redshift and applies to users in Van | Redshift, VAN|
+| s3_to_redshift.py  |  Moves files from S3 to Redshift| Redshift, S3|
+| s3_to_s3.py | Get files from vendor s3 bucket and moves to own S3 bucket | S3 |
```

Comment:
I think "vendor" to "own" is maybe what the original script was for, but I think the script can be used for transferring data between any two buckets. I think we can generalize this to just "source" bucket to "target" bucket or something.